### PR TITLE
graceful handling of unset settings

### PR DESCRIPTION
**Pull request by stefanfoulis**
Use blank defaults for `MULTISITE_CMS_URLS`, `MULTISITE_CMS_ALIASES` and
`MULTISITE_CMS_FALLBACK` to minimise required configuration

Also: Always `patch_vary_headers` on `Host`

### CODE REVIEW

File: djangocms_multisite/middleware.py [hunk 1]
```diff
@@ -9,20 +9,29 @@
 
 class CMSMultiSiteMiddleware(object):
     def process_request(self, request):
+        MULTISITE_CMS_URLS = getattr(settings, 'MULTISITE_CMS_URLS', {})
```
Comment (Chive) [inline djangocms_multisite/middleware.py]: I'd prefer to have these 3 constants at either class or even module level to decrease the amount of those lookups

**Comment by Chive** [comment]
I'd prefer to have these 3 settings loaded at either class or even module level to decrease the amount of those lookups.

Apart from that: LGTM

**Comment by stefanfoulis** [comment]
Lookups in the method are on purpose, because it makes testing easier. Performance impact should be negligible.

**Comment by yakky** [comment]
@stefanfoulis what's the status of this one? Is complete on your side?

**Comment by Chive** [comment]
@stefanfoulis but testing is done now no? I see no reason for keeping those there..

**Comment by Chive** [comment]
@stefanfoulis but testing is done now no? I see no reason for keeping those there..

**Pull request merged**